manifests/ocp/02-serviceaccount-tigera-operator.yaml

```diff
@@ -3,5 +3,5 @@ kind: ServiceAccount
 metadata:
   name: tigera-operator
   namespace: tigera-operator
-imagePullSecrets:
+imagePullSecrets: 
```

Comment:
We should avoid the extra space at the end of this line as it fails yaml linting, which some users run on our manifests.